### ISSUE TITLE
unexpected `end', expecting end-of-input

### ISSUE THREAD

**Issue by staleo** [kddnewton/syntax_tree#14]
```
rails -v
> Rails 7.0.1
ruby -v
> ruby 2.7.2p137 (2020-10-01 revision 5445e04352) [x86_64-linux]
``` 
I do `gem add syntax_tree`, than `rails s` and receive a ton of the following (and similar) in logs:

`/home/staleo/.rbenv/versions/2.7.2/lib/ruby/gems/2.7.0/gems/syntax_tree-1.2.0/lib/syntax_tree.rb:14006: syntax error, unexpected end', expecting end-of-input`

Any ideas how to solve this? Thanks!

**Comment by kddnewton** [comment]
Yeah, I'm using argument forwarding which was available in `2.7.3`. Sorry I haven't tested on old ruby versions yet. Can you try on `2.7.3` and see if it works?

**Comment by staleo** [comment]
Hi Kevin, right, switched to 3.0, and it's back to business now.
